crates/legacy/src/service.rs

```diff
@@ -1540,6 +1552,9 @@ mod test {
         HandleQuorumEventError, HandleReceivedTxns, ProxyGlobalState,
     };
 
+    /// A const number on `max_tx_len` to be used consistently spanning all the tests
+    const TEST_MAX_TX_LEN: u64 = 20;
```

Comment:
Double-checking how we get `20`--is it length `4` + overhead `16`? Can you add a comment here?

Reply:
Updated to 1 as current length on TestTransaction is 1.